app/components/CommentForm/index.js

```diff
@@ -69,7 +70,14 @@ const CommentForm = ({
         );
       }}
     >
-      {({ handleSubmit, pristine, submitting, form }) => {
+      {({
+        handleSubmit,
+        pristine,
+        submitting,
+        form,
+        // $FlowFixMe
```

Comment:
Was this too difficult to type? Our webapp has already enough technical debt 😬

Reply:
yes